android/app/build.gradle

```diff
@@ -37,6 +37,17 @@ dependencies {
     testCompile 'junit:junit:4.12'
     compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
     compile 'org.jsoup:jsoup:1.10.2'
+
+    // Retrofit
+    compile 'com.squareup.retrofit2:retrofit:2.2.0'
+    compile 'com.squareup.retrofit2:converter-scalars:2.2.0'
```

Comment:
It can convert a response to `String`, I fetch a html page as string and parse it